getting_started/README.md

```diff
@@ -0,0 +1,241 @@
+
+It's also helpful to checkout the [OpenTracing Tutorial for C# (opentracing-tutorial)][opentracing-tutorial] first or afterwards, which also uses [jaeger-client-csharp] as OpenTracing implementation.
+
+For more details on the C# API of OpenTracing, have a look at [opentracing-csharp]. But most of the API is already handled in [opentracing-tutorial].
+
+Having followed the [gRPC Quickstart for C#](https://grpc.io/docs/quickstart/csharp.html) also helps understanding.
+
+## Installation
+
+The tutorial is using CNCF Jaeger (https://github.com/jaegertracing/jaeger) as the tracing backend, which uses [Docker](https://www.docker.com/get-docker). You will still be able to follow most of the tutorial, even without Docker. So don't worry if you can't or don't want to install it.
+
+For running the code, you will need the [.NET Core SDK](https://www.microsoft.com/net/download). The code can be written with any text editor and run from the command line.
+
+For easier development, use the free IDE [Visual Studio Code](https://code.visualstudio.com/Download?wt.mc_id=DotNet_Home) available for Windows, Linux and macOS. For full language support including smart code completion and debugging, get the [C# extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp) for Visual Studio Code. Alternatively, if you already have Visual Studio 2017 or higher, .NET Core SDK is already included.
+
+## Download the example
+
+The example is based on an updated version of the [gRPC Quickstart example for .NET Core](https://github.com/grpc/grpc/tree/master/examples/csharp/helloworld-from-cli). You can not use the example code from `grpc/grpc` without updating the NuGet packages and upgrading the code from `.netcoreapp1.0` to `.netcoreapp2.0`.
+
+You�ll need a local copy of the example code to work through this quickstart. Download the example code from our Github repository (the following command clones the entire repository, but you just need the examples for this quickstart):
+
+```bash
+> # Clone the repository to get the example code:
+> git clone https://github.com/opentracing-contrib/csharp-grpc
+> cd csharp-grpc/getting_started
+```
+
+## Run the example
+
+Either open the solution `Greeter.sln` with Visual Studio or Xamarin Studio or just build it using the .NET Core SDK. 
+It's easier to use the command line since we need to have two running instances.
+
+### Run the server
+
+```bash
+> cd GreeterServer
+> dotnet run
+```
+
+### Run the client (in another terminal)
+
+```bash
+> cd GreeterClient
+> dotnet run
+```
+
+## Install NuGet packages
+
+Add the following NuGet packages as dependencies to the shared `GreeterShared` project. This can be done through the .NET Core SDK using the following commands.
+
+```bash
+> dotnet add GreeterShared package Microsoft.Extensions.Logging.Console
+> dotnet add GreeterShared package OpenTracing.Contrib.Grpc
+> dotnet add GreeterShared package Jaeger
+```
+
+## Get the tracer ready
```

Comment:
so what is the objective of this example - a mini-tutorial or just a sample code? Is this something that people can run out of the box, or do they need to make changes described below? I wonder if it might be better to just provide a runnable code with comments in the code, and have the README provide just the high-level explanation of the structure / navigation.

Reply:
The code is based on the official gRPC example and enhances it with tracing. It's structured in a step-by-step way and also let's the use experience every change he makes. When having all the code there, it would loose that point. It could just make it like we have it in `opentracing-tutorial` that I add `excercise` template and `solution`. But since it's only one additional method and two changes (plus some declarations), I thought that would be easier to follow. Especially since it's shown in a demo.